src/db/repository/postgres/WishCardsRepository.ts

```diff
@@ -0,0 +1,73 @@
+import { Kysely, UpdateObject } from 'kysely';
+
+import { DB, Wishcards } from '../../types/generated/database';
+
+export type WishcardsUpdateParams = Omit<
+	UpdateObject<DB, 'wishcards'>,
+	'id' | 'created_at' | 'updated_at' | 'agency_id' | 'child_id'
+>;
+export type WishcardsCreateParams = Omit<
+	Wishcards,
+	'id' | 'created_at' | 'updated_at' | 'deleted_at' | 'status'
+>;
+
+export default class WishcardsRepository {
+	constructor(private readonly database: Kysely<DB>) {}
+
+	async create(createParams: WishcardsCreateParams) {
+		const wishcard = await this.database
+			.insertInto('wishcards')
+			.values(createParams)
+			.returningAll()
+			.executeTakeFirstOrThrow();
+
+		return wishcard;
+	}
+
+	async getAll() {
+		return await this.database.selectFrom('wishcards').selectAll().execute();
+	}
```

Comment:
we can omit all the `return await ...` statements and the async declaration
we return the db promise without any error handling since this is part of business logic